recipes/ghc/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ghc" %}
+{% set version = "8.2.2" %}
+{% set sha256 = "bb8ec3634aa132d09faa270bbd604b82dfa61f04855655af6f9d14a9eedc05fc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: https://downloads.haskell.org/~ghc/{{ version }}/{{ name }}-{{ version }}-src.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - xz 5.2.*
+    - autoconf
+    - automake
+    - libtool
+    - toolchain
+    - gmp >=5.0.1,<7
+  run:
+    - xz 5.2.*
+    - gmp >=5.0.1,<7
+
+test:
+  commands:
+    - ghc --help
+
+about:
+  home: https://haskell.org/ghc/
+  license: GPL-like
+  license_family: GPL
```

Comment:
Think it is `BSD 3-Clause`.

ref: https://www.haskell.org/ghc/license.html

Reply:
Should add the license included in the recipe is also `BSD 3-Clause`.

Reply:
thanks, fixed :-)